### DISCUSSION TITLE
More detailed documentation support is required.

### DISCUSSION THREAD

**Discussion by WCKYWCKF** [AvaloniaUI/AvaloniaEdit#439] [General]
More detailed documentation will help newcomers learn how to create interesting programs using AvaloniaEdit, and learning how to use AvaloniaEdit through a demo application and cross-reference to the source code will take more time than reading a detailed document.

I found a document in http://avalonedit.net/documentation/, but it's too old.

Can you add it to this year's to-do?

**Comment by maxkatz6** [comment]
> http://avalonedit.net/documentation/

That's a doc for original implementation, not for Avalonia fork.
Although it's very similar for the most part.

> Can you add it to this year's to-do?

Not planned.
